trieste/acquisition/sampler.py

```diff
@@ -61,33 +61,77 @@ def sample(self, at: TensorType) -> TensorType:
         """
 
 
-class DiscreteThompsonSampler(DiscreteSampler):
+class ThompsonSampler(Sampler, ABC):
     r"""
-    This sampler provides approximate Thompson samples of the objective function's
+    A :class:`ThompsonSampler` samples either the minimum values or minimisers of a function
+    modelled by an underlying :class:`ProbabilisticModel` across a  discrete set of points.
```

Comment:
If we're using US spelling :-)

```suggestion
    modeled by an underlying :class:`ProbabilisticModel` across a  discrete set of points.
```